src/components/DpSelect/DpSelect.vue

```diff
@@ -67,6 +68,12 @@ export default {
       default: ''
     },
 
+    dataCy: {
+      type: String,
+      required: false,
+      default: 'accordionToggle'
```

Comment:
As the default, i'd chose something that just describes the element within the DpSelect component:

```suggestion
      default: 'selectElement'
```